commit a0d7a43e5234305bd6f1de5909d4ce5d7e1ea864
Author: Robert Dodier <robert_dodier@users.sourceforge.net>
Date:   Mon Oct 11 22:40:13 2021 -0700

    Apply patch #80: "Build a FASL library, needed by SageMath"
    
    This modifies maxima.system for ECL only.

diff --git a/src/maxima.system b/src/maxima.system
index d954bc2b9..525fb778e 100644
--- a/src/maxima.system
+++ b/src/maxima.system
@@ -75,6 +75,11 @@
 			     ;; Convert dir/foo.fas to dir/foo.o
 			     (make-pathname :type "o" :defaults p))
 			 files)))
+	(c::build-fasl "binary-ecl/maxima" :lisp-files obj
+			  :ld-flags
+			  (let ((x (symbol-value (find-symbol "*AUTOCONF-LD-FLAGS*"
+							      (find-package "MAXIMA")))))
+			    (if (and x (not (string= x ""))) (list x))))
 	(c::build-program "binary-ecl/maxima" :lisp-files obj
 			  :ld-flags
 			  (let ((x (symbol-value (find-symbol "*AUTOCONF-LD-FLAGS*"
